lcurtis4/my-meals
# Login & Registration

## What it does
This code establishes both the login and registration form for the user. 

## Testing 

1. Pull the new code down 
2. Serve it up
3. Proceed to register account and see that the info is added to the database.json file
4. See that new user added is able to login from email account
